certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2934 @@
+        "id": {
+          "format": "uuid",
+          "x-ms-summary": "KeyDate Id",
+          "description": "Key Date unique identifier",
+          "type": "string"
+        },
+        "attributes": {
+          "$ref": "#/definitions/KeyDateAttributes"
+        },
+        "sourceType": {
+          "$ref": "#/definitions/SourceType"
+        }
+      }
+    },
+    "KeyDateResponseListResponse": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KeyDateResponse"
+          }
+        }
+      }
+    },
+    "KeyDateResponseResponse": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "$ref": "#/definitions/KeyDateResponse"
+        }
+      }
+    },
+    "MeResponse": {
+      "description": "User details",
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "uuid",
+          "x-ms-summary": "User Id",
+          "description": "User unique identifier",
+          "type": "string"
+        },
+        "logonAccount": {
+          "description": "User Logon Account",
+          "x-ms-summary": "Logon Account",
+          "type": "string"
+        },
+        "email": {
+          "description": "User Email",
+          "x-ms-summary": "Email",
+          "type": "string"
+        },
+        "token": {
+          "$ref": "#/definitions/TokenInfo"
+        }
+      }
+    },
+    "MeResponseResponse": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "$ref": "#/definitions/MeResponse"
+        }
+      }
+    },
+    "ProjectAttributes": {
+      "x-ms-summary": "Project fields",
+      "description": "Project fields (e.g. {\"Name\": \"My Name\", \"Description\": \"TBD\"})",
+      "type": "object",
+      "properties": {},
+      "additionalProperties": {}
+    },
+    "ProjectResponse": {
+      "description": "Project details",
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "uuid",
+          "x-ms-summary": "Project Id",
+          "description": "Unique Project Identifier",
+          "type": "string"
+        },
+        "attributes": {
+          "$ref": "#/definitions/ProjectAttributes"
+        }
+      }
+    },
+    "ProjectResponseListResponse": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "array",
+          "x-ms-summary": "Projects",
+          "description": "List of projects",
+          "items": {
+            "$ref": "#/definitions/ProjectResponse"
+          }
+        }
+      }
+    },
+    "ProjectResponseResponse": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "$ref": "#/definitions/ProjectResponse"
+        }
+      }
+    },
+    "RequestHeaderInfo": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "value": {
+          "type": "string"

```

Comment:
@vsorokin-ppmexpress can you please tell me what headers are you going to use here?

Reply:
I don't use these headers with Power Automate, but there is such an option to add custom headers to the request that is coming from our API service to the URL that was set in Webhook Create action.